aws_cloudtrail_rules/aws_cloudtrail_stopped.py

```diff
@@ -10,11 +12,15 @@ def rule(event):
 
 
 def dedup(event):
-    # Merge on the specific CloudTrail
+    # Merge on the CloudTrail ARN
     return event['requestParameters'].get('name')
 
 
 def title(event):
     trail_arn = event['requestParameters'].get('name')
-    return 'CloudTrail {} Has Been Disabled with {}'.format(
-        trail_arn, event['eventName'])
+    if event['eventName'] == 'DeleteTrail':
+        action = 'deleted'
+    elif event['eventName'] == 'StopLogging':
+        action = 'stopped'
+    return 'CloudTrail [{}] in account [{}] has been {}'.format(
+        trail_arn, lookup_aws_account_name(event['recipientAccountId']), action)
```

Comment:
If neither condition is true, you will get a runtime exception:

```
UnboundLocalError: local variable 'action' referenced before assignment
```

I know this may not be possible right now with the rule, but still better to have a default value rather than risk an exception. Set `action = '(unknown)'` or some other default before the if statement on line 21